cluster/update-storage-objects.sh

```diff
@@ -59,7 +61,12 @@ for resource in "${resources[@]}"
 do
   for namespace in "${namespaces[@]}"
   do
+    # If get fails, assume it's because the resource hasn't been installed in the apiserver.
+    # TODO hopefully we can remove this once we use dynamic discovery of gettable/updateable
+    # resources.
+    set +e
```

Comment:
Ok, it looks like the only way to really do this would be something like this:

```
mkfifo somefifo
instances=( $("${KUBECTL}" get "${resource}" --namespace="${namespace}" -o go-template="{{range.items}}{{.metadata.name}} {{end}}" 2>somefifo))
```

Then, if you read somefifo and it's either empty or it contains "the server doesn't have a resource type", then we could proceed. Otherwise, we should consider it an error. We'd need to have a trap to remove the fifo on exit, as well. Do you want me to proceed with this?